tests/command-line/clean/pass.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+WAKE="${1:+$1/wake}"
+"${WAKE:-wake}" -q --init .
+"${WAKE:-wake}" -q -x 'write "bug" ""'
```

Comment:
Specifically, this doesn't *directly* create a `Path "."`, but it calls `mkdir` to create that (current) directory so it covers what we're looking for.